lib/msf/core/rpc/v10/rpc_health.rb

```diff
@@ -0,0 +1,35 @@
+# -*- coding: binary -*-
+module Msf
+module RPC
+class RPC_Health < RPC_Base
+
+  # Returns whether the service is currently health and ready to accept
+  # requests. This endpoint is not authenticated.
+  #
+  # @return [Hash]
+  # @example Here's how you would use this from the client:
+  #  rpc.call('health.check')
+  def rpc_check_noauth
```

Comment:
Originally I planned to just call the existing rpc_core version endpoint, but that requires auth. Opted for a separate mechanism for checking that everything's healthy without leaking information